macros/sql/union.sql

```diff
@@ -1,10 +1,11 @@
-{% macro union_tables(tables, column_override=none, exclude=none, source_column_name=none) -%}
+{% macro union_tables(tables, column_override=none, include=none, exclude=none, source_column_name=none) -%}
```

Comment:
let's get rid of this jankiness while we're here :)
```suggestion
{% macro union_tables(tables, column_override=none, include=[], exclude=[], source_column_name=none) -%}
```